_pages/en/strategy-build-vs-buy.md

```diff
@@ -193,7 +283,7 @@ Software options in this niche market are scarce due to the unique mandate of ES
 Change probability is high as those capabilities change as much as regulation changes.
 The impact of such changes has a ripple effect on the whole organization and affects ESDC’s reputation to keep up with citizen demands while maintaining its fiduciary responsibility.
 
-### Build
+### Build<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).

Reply:
Good point, need to reassess this appendix to see how to properly explain what the overall situation is though.